test/nn/test_pooling.py

```diff
@@ -691,7 +708,6 @@ def helper(n, c, h, w, kernel_size, stride=None,
         # ROCm 16GB MI25 hits OOM error. Clear caching allocator prior to running large subtest.
         if TEST_WITH_ROCM and 'cuda' in device:
             torch.cuda.empty_cache()
-        helper(200, 512, 28, 28, 2)
```

Comment:
This case and the below case threw:
```python
RuntimeError: element 0 of tensors does not require grad and does not have a grad_fn
```
hence removed.

Reply:
Wait, was it passing before your PR? I don't see anything special about this input.

Reply:
Yes, this case was passing.

That time there were some memory errors in my system. I think I was not right about removing this. I resorted to the original. Thanks, Natalia for pointing it out!